src/ansys/additive/core/single_bead.py

```diff
@@ -193,8 +244,36 @@ def __eq__(self, __o: object) -> bool:
     def __repr__(self):
         repr = type(self).__name__ + "\n"
         repr += self._df.to_string()
+        repr += (
+            "\n" + "grid_full_thermal_sensor_file_output_path: " + str(self.thermal_history_output)
+        )
         return repr
 
+    @property
+    def thermal_history_output(self) -> str:
+        """Return path to the VTK file(s) containing the thermal history of the simulation."""
+        return self._thermal_history_output
+
+    @thermal_history_output.setter
+    def thermal_history_output(self, value: str | os.PathLike | None):
+        """Set the path to the VTK file(s) containing the thermal history of the simulation."""
+        if value is None:
+            self._thermal_history_output = None
+            return
+        self._thermal_history_output = pathlib.Path(value)
+        if not os.path.exists(self._thermal_history_output):
+            os.makedirs(self._thermal_history_output)
+
+    def _unarchive_grid_full_thermal_sensor_file(self, thermal_history_vtk: bytes):
+        """Unarchive the grid full thermal sensor file output."""
+        self.thermal_history_output = self.CONST_GRID_FULL_THERMAL_HISTORY_OUTPUT_PATH
```

Comment:
This path is wrong. It needs to be <user data path>/<simulation id>/CONST_GRID_FULL_THERMAL_HISTORY_OUTPUT_PATH. See what we do in `_simulate_thermal_history()`. You'll want to add `out_dir` to the parameter list for `SingleBeadSummary.__init__()`, just like `ThermalHistorySummary.__init__()`. In `simulate()`, you can set the `out_dir` value to `USER_DATA_PATH`.

With what you have, if the user ran two single bead simulations with thermal-history, the second simulation would overwrite the thermal history results from the first one.